browser-extension/mv2/tests/specs/modules/PreDefinedFunctions.spec.js

```diff
@@ -109,25 +81,52 @@ describe("PreDefined Functions: ", function () {
 
     it("should replace rq_decrement usages with value decremented by 1", function () {
       expect(F.eval("rq_decrement(10)")).toBe("9");
-      expect(F.eval("http://www.example.com/?page=rq_decrement(10)")).toBe(
-        "http://www.example.com/?page=9"
-      );
+      expect(F.eval("http://www.example.com/?page=rq_decrement(10)")).toBe("http://www.example.com/?page=9");
     });
 
     it("should replace rq_decrement usages with value decremented by k", function () {
       expect(F.eval("rq_decrement(10,3)")).toBe("7");
-      expect(F.eval("http://www.example.com/?page=rq_decrement(10,3)")).toBe(
-        "http://www.example.com/?page=7"
-      );
+      expect(F.eval("http://www.example.com/?page=rq_decrement(10,3)")).toBe("http://www.example.com/?page=7");
     });
 
     it("should replace rq_decrement usages with value decremented by k ignoring spaces before/after comma", function () {
       expect(F.eval("rq_decrement(10, 3)")).toBe("7");
       expect(F.eval("rq_decrement(10 ,3)")).toBe("7");
       expect(F.eval("rq_decrement(10 , 3)")).toBe("7");
-      expect(F.eval("http://www.example.com/?page=rq_decrement(10, 3)")).toBe(
-        "http://www.example.com/?page=7"
-      );
+      expect(F.eval("http://www.example.com/?page=rq_decrement(10, 3)")).toBe("http://www.example.com/?page=7");
+    });
+  });
+
```

Comment:
Below this line is the actual addition in this file. Rest are linting fixes pushed automatically with commit hook.